XenAdmin/Wizards/ImportWizard/ImportWizard.cs

```diff
@@ -582,7 +582,7 @@ private IEnumerable<Tuple> GetSummaryXva()
         {
             var temp = new List<Tuple>();
             temp.Add(new Tuple(Messages.FINISH_PAGE_VMNAME, m_pageXvaStorage.ImportedVm.Name()));
-            temp.Add(new Tuple(Messages.FINISH_PAGE_TARGET, m_pageXvaHost.SelectedHost == null ? m_pageXvaHost.SelectedConnection.Name : m_pageXvaHost.SelectedHost.Name()));
+            temp.Add(new Tuple(Messages.FINISH_PAGE_TARGET_FOR_VM, m_pageXvaHost.SelectedHost == null ? m_pageXvaHost.SelectedConnection.Name : m_pageXvaHost.SelectedHost.Name()));

```

Comment:
This one is still needed because XVA import does set the home server, it's only OVF that doesn't set it.